### ISSUE TITLE
undefined is not an object (evaluating '_reactNative.Animated.Text.propTypes.style') & Animated.Text.propTypes.style problem on react-native-material-textfield in RN 0.63.4

### ISSUE THREAD

**Issue by ravinderKatta** [n4kz/react-native-material-textfield#317]
### > facing issues in RN 0.63.4 and can't even run after installing this library (react-native-material-textfield) pls fix it ASAP.
>**undefined is not an object (evaluating '_reactNative.Animated.Text.propTypes.style') & Animated.Text.propTypes.style problem on react-native-material-textfield in RN 0.63.4**
> who are facing same issue, for temporary purpose you can use [(npm install rn-material-ui-textfield)](url)

**Comment by midrizi** [comment]
Any update on this?

**Comment by gabrieldonadel** [comment]
Hi @midrizi and @ravinderKatta, unfortunately this is a known bug in `react-native-material-textfield` and this lib appears to be abandoned. 
I suggest you using this fork [rn-material-ui-textfield](https://www.npmjs.com/package/rn-material-ui-textfield) that has fixed this and some other bugs. 

If you want to give it a try just run

```
npm install --save rn-material-ui-textfield

**Comment by midrizi** [comment]
Yes you're right, I did give that lib a try,  but it didn't have TS support, I did fix the issues I had with this lib, seems to also have other issues as well, with animations and some other stuff, looks like the lib you suggested to be the best option for now.

Thanks!

**Comment by klein141** [comment]
If you're searching for other alternatives I found [React Native Paper](https://callstack.github.io/react-native-paper/index.html) to work for my use case
